xbmc/pvr/windows/GUIWindowPVRSearch.cpp

```diff
@@ -241,6 +240,14 @@ bool CGUIWindowPVRSearch::OnMessage(CGUIMessage &message)
       }
     }
   }
+  else if (message.GetMessage() == GUI_MSG_NOTIFY_ALL)
+  {
+    if (message.GetParam1() == GUI_MSG_UPDATE_SOURCES)
+    {
+      // removable drive connected/disconnected. base class triggers a window content refresh, which makes no sense for pvr search window.
+      return true;
+    }
+  }
```

Comment:
@phil65 might be a good idea for v18. at this point for v17 i only want to make changes i'm pretty sure not to break something.